src/test/java/tec/uom/se/format/SymbolMapTest.java

```diff
@@ -52,7 +52,7 @@
    */
   @Test
   public void testSymbolMap() {
-    SymbolMap symbols = SymbolMap.of(ResourceBundle.getBundle(BUNDLE_NAME, Locale.getDefault()));
+    SymbolMap symbols = SymbolMap.of(ResourceBundle.getBundle(BUNDLE_NAME, Locale.ROOT));

```

Comment:
The biggest issue here is that you were using `Locale.getDefault()` to test `messages.properties`, but `getDefault()` means "give me the default locale being used by the current machine", and not a "root" locale that would get the default `messages.properties`.